apiserver/facades/controller/firewaller/firewaller_base_test.go

```diff
@@ -180,13 +179,6 @@ func (s *firewallerBaseSuite) testInstanceId(
 		InstanceId(context.Context, params.Entities) (params.StringResults, error)
 	},
 ) {
-	// Provision 2 machines first.
-	err := s.machines[0].SetProvisioned("i-am", "", "fake_nonce", nil)
-	c.Assert(err, jc.ErrorIsNil)
-	hwChars := instance.MustParseHardware("arch=s390x", "mem=4G")
-	err = s.machines[1].SetProvisioned("i-am-not", "", "fake_nonce", &hwChars)
-	c.Assert(err, jc.ErrorIsNil)
```

Comment:
Was this just not required? Probably worth explaining why we can delete this.

Reply:
We can remove this because we are now reading from dqlite, in this case these are mocked in `TestInstanceId()` for example.